src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -736,47 +715,47 @@ class Settings extends Component {
 		}
 
 		let menuItems = cookies.get('loggedIn')?
-			<Menu
-				style={{margin: 'auto', width: '100%', }}
+			<Menu className="settings-list"
 				onItemTouchTap={this.loadSettings}
+				selectedMenuItemStyle={{color: 'rgb(66, 133, 244)'}}
```

Comment:
This color is being used multiple times, can this be fetch from a variable name? That way a change somewhere will change all of them